src/easyreflectometry/project.py

```diff
@@ -49,8 +41,8 @@ def __init__(self):
         self._path_project_parent = Path(os.path.expanduser('~'))
         self._models = ModelCollection(populate_if_none=False, unique_name='project_models')
         self._materials = MaterialCollection(populate_if_none=False, unique_name='project_materials')
-        self._calculator = None
-        self._minimizer: AvailableMinimizers = None
+        self._calculator = CalculatorFactory()
+        self._minimizer = AvailableMinimizers.LMFit_leastsq
```

Comment:
Maybe the default minimizer should be defined somewhere central? Top of the file as `DEFAULT_MINIMIZER`?
So you can avoid duplication and help with maintenance.

Reply:
Good point. Changed